lms/static/sass/course/_info.scss

```diff
@@ -56,6 +56,115 @@ div.info-wrapper {
       font-style: normal;
     }
 
+    div.upgrade-banner {
+      // This banner uses the Pattern Library's defined variables
+      @include border-left(0px);
+
+      border: 1px solid $uxpl-gray-background;
```

Comment:
Don't use colors directly in the Sass. There's already a ``$lms-border-color`` which you can use.